src/components/RewardsCard/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { DotIcon } from '@/assets/images'
+
+import { CardGeneric } from '../CardGeneric'
+import Sticker from '../Sticker'
+import styles from './index.module.scss'
+
+export type RewardsCardProps = {
+  id: number
+  questId: number
+  reward: string
+  rewardImage: string
+  claimsLeft?: string | number
+  i18n?: {
+    claimsLabel?: string
+    claimsLeftLabel?: string
+  }
+}
+
+function RewardsCard({
+  rewardImage,
+  reward,
+  claimsLeft,
+  i18n = {
+    claimsLabel: 'Claims left',
+    claimsLeftLabel: 'Unlimited'
+  }
+}: RewardsCardProps) {
+  return (
+    <CardGeneric
+      image={rewardImage}
+      isReward
+      genericClassNames={{
+        body: styles.rewardsCard,
+        image: styles.rewardImage
+      }}
+      className={styles.cardBase}
+    >
+      <div className={styles.content}>
+        <div className={styles.stickers}>
+          <Sticker styleType="neutral" dimension="default">
+            {reward}
```

Comment:
The design changed, and now the sticker is the one called "filledStrong" neutral. if you do not put the type here, it will load the first of the list, I believe. 
![image](https://github.com/user-attachments/assets/86f74a5d-e2d8-4271-b2f4-8da4ab61175c)

Reply:
Cool, thanks for the heads up